c-vendor/vendor.sh

```diff
@@ -5,10 +5,7 @@ set -e
 # XXX: provide a nice declarative format
 # XXX2: reuse vendor.json and write some python3 code?
 if [ ! -d ./squashfuse ]; then
-    git clone https://github.com/snapcore/squashfuse
+    git clone https://github.com/vasi/squashfuse
 fi
-(cd squashfuse && git checkout 319f6d41a0419465a55d9dcb848d2408b97764f9)
+(cd squashfuse && git checkout 74f4fe86ebd47a2fb7df5cb60d452354f977c72e)
```

Comment:
could you leave a comment on this explaining that this is just tip/master as of Aug 30th 2021, so we aren't left in the future wondering if there was a specific reason we used this commit? Context around specific things like this tends to get lost leading to a rabbit hole of "why did I pick this specific commit 5 years ago" etc.